tests/test_exec.py

```diff
@@ -62,11 +64,71 @@ def test_exec_not_exists():
 
 def test_exec_detach_not_exists():
     return test_exec_not_exists_helper(False)
+
+NONKM_OK_CONF = "var/lib/krun/nonkm_exec_ok_config"
+
+def test_exec_non_km_payload_helper(execpath, nonkmok):
+    # only run these tests for krun
```

Comment:
pls add docustring with info on the function (for all new functions, no matter if it's the case for old ones)

Reply:
Will add.